### PR TITLE
<couldflared>: Add parameter protocol to startup

### PR DESCRIPTION
**Pull request by alex9434**
Maintainer: @1715173329 
Compile tested: OpenWrt Snapshot Arm64
Run tested: see above

Description:
Currently cloudflared on OpenWrt is not build with the toolchain that is required by the authors. Therefore the QUIC functionality is broken. At startup, the tool unsuccessfully tries for several minutes to establish a QUIC connection before falling back to the slower http2 connection. There currently seems no plan to switch the build for OpenWrt to the required toolchain to enable the functionality.

To mitigate the issue, I  propose to add the parameter "protocol" to the startup script with the default value of http2 to avoid the delay of several minutes before the tunnel is established. With this parameter to tool establishes the slower http2 connection immediately without first trying for a QUIC connection.

### CODE REVIEW

**Comment by 1715173329** [comment]
Please format your commits according to https://openwrt.org/submitting-patches.

File: net/cloudflared/files/cloudflared.init [hunk 1]
```diff
@@ -26,6 +26,7 @@ start_service() {
 
 	append_param_arg "config" "/etc/cloudflared/config.yml"
 	append_param_arg "origincert" "/etc/cloudflared/cert.pem"
+	append_param_arg "rotocol"
```
Comment (tiagogaspar8) [inline net/cloudflared/files/cloudflared.init:29]: misspell here for protocol

**Comment by 1715173329** [comment]
... and missing PKG_RELEASE bump.

File: net/cloudflared/files/cloudflared.init (hunk 1 above)
Comment (alex9434) [reply net/cloudflared/files/cloudflared.init:29]: sorry! Of course should read "protocol"

**Comment by 1715173329** [comment]
ping

**Pull request closed without merging**